components/admin_console/admin_definition.jsx

```diff
@@ -4808,17 +4807,10 @@ const AdminDefinition = {
                         defaultMessage='deprecated'
                     />
                 ),
-                shouldDisplay: (license) => license.IsLicensed && license.OpenId === 'true',
+                shouldDisplay: () => true,
```

Comment:
Rather than making this return true, can we just remove it and it's usage?

https://github.com/mattermost/mattermost-webapp/blob/04b35606c320075879058b0653715c25d0077baa/components/admin_console/admin_sidebar/admin_sidebar.tsx#L204

Reply:
Let's remove this tag when removing 'deprecation' label. Removed this function call for now.